libs/platform/unit/platform_unit_test.cpp

```diff
@@ -21,6 +21,12 @@
 #include "ebpf_xdp_program_data.h"
 #include "ebpf_state.h"
 
+extern "C"
+{
+    extern ebpf_helper_function_prototype_t* ebpf_core_helper_function_prototype;
+    extern uint32_t _ebpf_core_helper_functions_count;
```

Comment:
```suggestion
    extern uint32_t ebpf_core_helper_functions_count;
```
Also, these extern definitions should be in a .h file instead of the .cpp file.